integration_tests/pyproject.toml

```diff
@@ -13,22 +13,19 @@ black = "^22.3.0"
 flake8-black = "^0.3.2"
 flake8-isort = "^4.1.1"
 pep8-naming = "^0.11.1"
-protobuf = "^3.13.0"
 grpcio = "^1.33.2"
 PyYAML = "^5.3.1"
 python-dateutil = "^2.8.1"
-web3 = "^5.20.1"
-eth-bloom = "^1.0.4"
+web3 = "^6.4.0"
```

Comment:
seems poetry update can't solve version correctly, while getting no [.egg-info](https://github.com/nix-community/poetry2nix/issues/1161) in newer web3.

Reply:
can we simply use the lock files from main branch?

Reply:
same error

Reply:
Pin to same nixpkgs version?